src/main/java/seedu/algobase/logic/commands/AddCommand.java

```diff
@@ -58,11 +58,12 @@ public CommandResult execute(Model model) throws CommandException {
         requireNonNull(model);
 
         if (model.hasProblem(toAdd)) {
-            throw new CommandException(MESSAGE_DUPLICATE_PROBLEM);
+            throw new CommandException(String.format(MESSAGE_DUPLICATE_PROBLEM, toAdd.getName()));
```

Comment:
`getName()` actually returns a `Name` instance. Probably it's better to use `toAdd.getName().fullName` just in case someday `toString` of `Name` is changed.